pandas/_libs/tslibs/period.pyx

```diff
@@ -1723,6 +1729,7 @@ cdef class _Period(PeriodMixin):
         int64_t ordinal
         PeriodDtypeBase _dtype
         BaseOffset freq
+        bint is_period
```

Comment:
Do we need this?

Reply:
I think we don't need this. I rolled back my changes.